cli/src/main.rs

```diff
@@ -96,7 +100,7 @@ pub fn main() {
 
         if !std::str::from_utf8(&git_log.stdout)
             .expect("git log stdout error")
-            .contains("d50ce9d618c1a2adc478ffbc3ad281512fbb0c15")
+            .contains("e8677475da2676fcfec57615de6330a7cb542cc1")
```

Comment:
I've just been bumping this up as time goes on, even in its own commit to bring it up to the latest `dev` - of course if this doesn't get incremented it won't stop working, but it does 'bake in' the idea it must be in a `zingolib` (or forked) repo.

Reply:
I didn't answer your question. I am not sure because branch names are just labels - the commit hash seemed more reliable. We can definitely revisit this though.